packages/services/src/services/Analytics/Analytics.js

```diff
@@ -214,6 +214,76 @@ class AnalyticsAPI {
       }
     }
   }
+
+  /**
+   * Sends video player metrics data
+   *
+   * @param {string} data event data from the video player
+   */
+  static videoPlayerStats(data) {
```

Comment:
We should include documentation on what data object structure is expected, either in the JSDocs or README?

Reply:
yep will do